.misc/deploy.sh

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env bash
+
+# Automatic deployer script
+# This script is meant to be run on solar.coala.io!
+
+WORKDIR=/opt/solar
+
+function log {
+    echo "$(date -u "+%b %Y %T") $@"
+}
+
+function update_instance {
+    instance=$1
+    full_path="$WORKDIR/$instance"
+
+    if [ -f "$full_path/docker-compose.yml" ]; then
+        pushd $full_path
+        log "Updating $instance ..."
+
+        docker-compose up --force-recreate --build -d
+
+        if [ $? -ne 0 ]; then
+           log "An error has occured while updating $instance!"
```

Comment:
Three spaces here, vs four elsewhere.
Should enable the relevant bear so reviewers dont need to be surprised by these things.